docs/install-alternative.md

```diff
@@ -17,19 +17,81 @@ Follow the instructions below, or [watch a video by Drupalize.me](https://youtu.
         # Install master branch as a git clone. Great for contributing back to Drush project.
         composer global require drush/drush:dev-master --prefer-source
 
-* Alternate way to install for all users via Composer:
-
-        COMPOSER_HOME=/opt/drush COMPOSER_BIN_DIR=/usr/local/bin COMPOSER_VENDOR_DIR=/opt/drush/7 composer require drush/drush:7
-
 * [Documentation for composer's require command.](http://getcomposer.org/doc/03-cli.md#require)
 * Uninstall with : `composer global remove drush/drush`
 
+Install Drush for all users via Composer
+------------
+If you need a common Drush install available for all users on a system, follow the composer install steps below.
+
+1. [Install Composer globally][composer install global].
+1. Create and/or navigate to a directory path for the single composer drush install. In this example we are going to install drush version 7.x
```

Comment:
Which version should we recommend here?  Drush 9 may go through periods of instability in the near future.  Drush 7 is two versions back now, and will probably get fewer backported bugfixes.  Maybe Drush 8 is best.